### PR TITLE
Updated the bom file SAP/HANA_2_00_067_v0003ms/HANA_2_00_067_v0003ms.yaml to include new links.

### PR DESCRIPTION
**Pull request by ShayakSarkar**
Description:

* SAP/HANA_2_00_067_v0003ms/HANA_2_00_067_v0003ms.yaml
  Updated the bom link https://softwaredownloads.sap.com/file/0020000000317632023 to this: https://softwaredownloads.sap.com/file/0020000000563052023

Validations:
The updated bom file is to be subjected to BOM Validator pipeline that will update check the validity of the new links.

### CODE REVIEW

File: SAP/HANA_2_00_067_v0003ms/HANA_2_00_067_v0003ms.yaml [hunk 1]
```diff
@@ -0,0 +1,74 @@
+---
+name:               'HANA_2_00_067_v0002ms'
```
Comment (KimForss) [inline SAP/HANA_2_00_067_v0003ms/HANA_2_00_067_v0003ms.yaml:2]:
```suggestion
name:               'HANA_2_00_067_v0003ms'
```

File: SAP/HANA_2_00_067_v0003ms/HANA_2_00_067_v0003ms.yaml [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+target:             'HANA 2.0'
+version:            1
```
Comment (KimForss) [inline SAP/HANA_2_00_067_v0003ms/HANA_2_00_067_v0003ms.yaml:4]:
```suggestion
version:            3
```

**Comment by praveennet** [comment]
Can we add TSG in this repo itself like we discussed?

**Comment by praveennet** [comment]
Please add more descriptive/ user friendly PR title and PR descriptions.

add what validations/ reviews done (or planned) to ensure this works

File: SAP/HANA_2_00_067_v0003ms/HANA_2_00_067_v0003ms.yaml (hunk 1 above)
Comment (ShayakSarkar) [reply SAP/HANA_2_00_067_v0003ms/HANA_2_00_067_v0003ms.yaml]: These changes have been made.

File: SAP/HANA_2_00_067_v0003ms/HANA_2_00_067_v0003ms.yaml (hunk 2 above)
Comment (ShayakSarkar) [reply SAP/HANA_2_00_067_v0003ms/HANA_2_00_067_v0003ms.yaml]: These changes have now been made.

**Review by KimForss — APPROVED** [review]

**Pull request merged**